hooks/queries/delegation/useCommittedVotesForDelegator.ts

```diff
@@ -14,7 +14,7 @@ export function useCommittedVotesForDelegator() {
   const { voting } = useContractsContext();
   const { getDelegationStatus, getDelegatorAddress } = useDelegationContext();
   const { roundId } = useVoteTimingContext();
-  const onError = useHandleError();
+  const { onError } = useHandleError({ isDataFetching: true });
```

Comment:
For queries, we pass `isDataFetching: true` so that the handler knows to show the generic data fetching error message.